releases.json

```diff
@@ -513,7 +513,7 @@
       "cwalk"
     ],
     "versions": [
-      "1.2.9-1"
+      "1.2.9-2"
```

Comment:
Don't patch the version, add a new one (first).

Reply:
You are fast 😆